projects/gstreamer/project.yaml

```diff
@@ -2,5 +2,6 @@ homepage: "https://gstreamer.freedesktop.org/"
 primary_contact: "gstreamer-security@lists.freedesktop.org"
 auto_ccs:
  - "bilboed@bilboed.com"
-
+ - "tim@centricular.com"
+ - "slomo@coaxion.net"
 
```

Comment:
Are these Google accounts? Asking because of https://github.com/google/oss-fuzz/blob/master/docs/faq.md#why-do-you-require-a-google-account-for-authentication

Reply:
Yes, they both are standalone google accounts